sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1388,6 +1390,28 @@ public List<CaseSelectionDto> getSimilarCases(CaseSimilarityCriteria criteria) {
 		return entries;
 	}
 
+	@Override
+    public List<CaseDataDto> getCasesForWhichVaccinationIsRelevant(VaccinationDto vaccinationDto) {
```

Comment:
There is a major flaw with this logic: It doesn't take relevance into account at all, but will just return all vaccinations of the person. However, we need to filter them by a) disease (because cases with other diseases should NOT be considered), and also need to take into account the vaccination date in relation to the case date, because cases that have been reported earlier than the vaccination should not be affected (see VaccinationService.isVaccinationRelevant).